frontend/src/turbo/turbo-global-listeners.ts

```diff
@@ -64,4 +64,13 @@ export function addTurboGlobalListeners() {
     activateFlashNotice();
     activateFlashError();
   });
+
+  document.addEventListener('turbo:before-morph-element', (event) => {
+    const element = event.target as HTMLElement;
+
+    // In case the element is an OpenProject custom dom element, morphing is prevented.
+    if (element.tagName.startsWith('OPCE-')) {
```

Comment:
According to this comment, https://github.com/opf/openproject/pull/17223#discussion_r1888464672 it would be advisable to use `toUpperCase` for comparing the tagname.
```suggestion
    if (element.tagName.toUpperCase().startsWith('OPCE-')) {
```